samples/FakeItEasy.Examples.VB/AssertingCalls.vb

```diff
@@ -13,4 +13,11 @@ Public Class AssertingCalls
 
         A.CallTo(Function() factory.CreateWithColor(A(Of String).Ignored)).MustHaveHappened()
     End Sub
+
+    Public Sub Asserting_ordered_function_calls()
+        Dim factory = A.Fake(Of IWidgetFactory)()
+
+        A.CallTo(Function() factory.CreateWithColor(A(Of String).Ignored)).MustHaveHappened().Then(

```

Comment:
VB doesn't require line termination characters, so can't always tell when a statement ends. Breaking the line before `.Then` confuses the compiler, so I hoisted the `(` (and preceding characters) up to this line. I am not sure what is more idiomatic in VB, but will switch to the [line confinuation character `_`](https://msdn.microsoft.com/en-us/library/ba9sxbw4.aspx?f=255&MSPPError=-2147217396) instead.